content/apps/quotas.md

```diff
@@ -2,64 +2,14 @@
 menu:
   main:
     parent: apps
-title: Quotas
+title: Application Quotas
```

Comment:
Do you mind if we change it to `Application Limits`? `Application Quotas` is a non standard term.